src/Config/SessionConfig.php

```diff
@@ -133,7 +133,18 @@ public function setStorageOption($storageName, $storageValue)
                 break;
         }
 
+        $sessionAlreadyStarted = false;
```

Comment:
I'd rename this to `$sessionRequiresRestart`, as it better conveys what needs to happen. This is particularly important when you come to line 144, as calling `session_start()` because `$sessionAlreadyStarted` seems like a mistake when just looking at the verbiage.

Reply:
I've updated variable to `sessionRequiresRestart`